html/index.html

```diff
@@ -550,6 +550,66 @@ <h1>Sampling and Sequencing Simulator</h1>
   throw new Error("failed to draw from cdf");
 }
 
+function daily_incidence_to_weekly_incidence(
```

Comment:
I'm confused about this function. What is `daily_incidence` exactly? It looks like a scalar, so what day does it represent? What is "effective" about `effective_incidence`? Why does it get divided by `growth_factor`?

Reply:
Oh wait, I think I get it. This is calculating the weekly incidence looking backwards from the current day. `daily_incidence` is the current daily incidence. The division gives you the daily incidence on each day going back in time. Is that right?

Reply:
That's right!  Added a comment in 087f449